llvm/test/tools/llvm-objdump/XCOFF/file-headers.test

```diff
@@ -0,0 +1,21 @@
+# Test the `--file-headers` option of llvm-objdump for XCOFF object files.  
```

Comment:
Nit: newer tools tests use `##` for comments, to distinguish from lit directives like `RUN:`